Formula/eccodes.rb

```diff
@@ -0,0 +1,37 @@
+class Eccodes < Formula
+  desc "Decode and encode messages in the GRIB 1/2 and  BUFR 3/4 formats"
+  homepage "https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home"
+  url "https://software.ecmwf.int/wiki/download/attachments/45757960/eccodes-2.7.0-Source.tar.gz"
+  sha256 "118f46cf6f800585580a5bc838128537ab0879073e9fcded49cd374e4c8d8e6a"
+  depends_on "cmake" => :build
+  depends_on "gcc" # for gfortran
+  depends_on "numpy"
+  depends_on "jasper" => :recommended
+  depends_on "libpng" => :optional
+
+  def install
+    # Fix "no member named 'inmem_' in 'jas_image_t'"
+    inreplace "src/grib_jasper_encoding.c", "image.inmem_    = 1;", ""
+
+    inreplace "CMakeLists.txt", "find_package( OpenJPEG )", ""
+
+    mkdir "build" do
+      args = std_cmake_args
+      args << "-DBUILD_SHARED_LIBS=OFF" if build.with? "static"
+
+      if build.with? "libpng"
+        args << "-DPNG_PNG_INCLUDE_DIR=#{Formula["libpng"].opt_include}"
+        args << "-DENABLE_PNG=ON"
+      end
+
+      system "cmake", "..", "-DENABLE_NETCDF=OFF", *args
+      system "make", "install"
+    end
+  end
+
+  test do
+    grib_samples_path = shell_output("#{bin}/codes_info -s").strip
+    system bin/"grib_ls", "#{grib_samples_path}/GRIB1.tmpl"
```

Comment:
let's assert something here beyond just a zero exit code

Reply:
I have unfortunately nothing else to assert here, as the contents of these files are from the ECMWF an can change at any time. It simply means that the library builds and the utilities are able to work on the files.